### PR TITLE
implemented child agencies markup endpoints

### PR DESCRIPTION
**Pull request by DFSko**
resolves happy-travel/agent-app-project#10

### CODE REVIEW

**Review by kirillta — APPROVED** [review]
Waiting for the formula update. So far so good

File: Api/Controllers/AgentControllers/AgencyMarkupController.cs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        }
+        
+        
+        /// <summary>
+        /// Creates markup policy.
+        /// </summary>
+        /// <param name="agencyId">Agency id</param>
+        /// <param name="settings">Markup settings</param>
+        /// <returns></returns>
+        [HttpPost("agency/agencies/{agencyId}/markups")]
```
Comment (mingazhev) [inline Api/Controllers/AgentControllers/AgencyMarkupController.cs:33]: maybe agency/child-agencies?

**Review by mingazhev — COMMENTED** [review]
Routes review needed

File: Api/Controllers/AgentControllers/AgencyMarkupController.cs (hunk 1 above)
Comment (DFSko) [reply Api/Controllers/AgentControllers/AgencyMarkupController.cs]: Fixed

**Review by mingazhev — APPROVED** [review]

**Pull request merged**